cdap-ui/app/cdap/services/__tests__/helpers.test.js

```diff
@@ -57,4 +57,10 @@ describe('Unit Tests for Helpers: "getArtifactNameAndVersion"', () => {
     expect(version).toBe(undefined);
   });
 
+  it('Usecase 7: Should return filename for name and version when filename is the version i.e 1.2.3.jar', () => {
```

Comment:
👏 👏 On making sure to add a unit test and fixing existing one based on the change. Kudos @itsanudeep